contrib/for-tests/volumes/nfs/run_nfs

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function start()
+{
+
+    # prepare /etc/exports
+    for i in "$@"; do
+        echo "$i *(rw,sync,no_root_squash,insecure,fsid=0,no_subtree_check)" >> /etc/exports
```

Comment:
Similar to previous comment, please explain in a comment why each of these options is required, or remove if not required.